gr-digital/lib/costas_loop_cc_impl.h

```diff
@@ -41,21 +41,21 @@ class costas_loop_cc_impl : public costas_loop_cc
      *  \param sample complex sample
      *  \return the phase error
      */
-    float phase_detector_8(gr_complex sample) const; // for 8PSK
+    inline float phase_detector_8(gr_complex sample) const; // for 8PSK
```

Comment:
I don't think this `inline` here has the desired effect. The `pahse_detector_*` functions are used only in the `workDefault` through the function pointer `d_phase_detector` so they are not inlined.

Reply:
I did not check the compiled code, but I believe that as well.